src/settings.ts

```diff
@@ -101,7 +101,7 @@ export class CopilotSettingTab extends PluginSettingTab {
         })
       )
       .addText((text) =>{
-        text.inputEl.type = "number";
+        text.inputEl.type = "decimal";
```

Comment:
Could you point me to the `decimal` type HTML specification? How does it behave compared to `number`?

Reply:
You're right. I can't and this PR should be closed. I was overly quick to suggest it.